clang/lib/3C/ConstraintResolver.cpp

```diff
@@ -473,13 +475,13 @@ CVarSet ConstraintResolver::getExprConstraintVars(Expr *E) {
           assert(CV.hasValue() && "Function without constraint variable.");
           /* Direct function call */
           if (FVConstraint *FVC = dyn_cast<FVConstraint>(&CV.getValue()))
-            ReturnCVs.insert(FVC->getReturnVar());
+            ReturnCVs.insert(FVC->getExternalReturn());
           /* Call via function pointer */
```

Comment:
For function pointers: Are the internal and external variables equated? (I expect that using the external one here makes sense, in any case.)